docs/scuttling.md

```diff
@@ -0,0 +1,74 @@
+## Intro
+
+Scuttling is a LavaMoat optional security feature, first introduced in [#360](https://github.com/LavaMoat/LavaMoat/pull/360).
+
+## Warning ⚠️
+
+This feature is experimental. This means that:
+1. Getting it to work on your LavaMoat protected application without breaking parts of it might require some effort, especially if your application loads JavaScript code outside of LavaMoat protection.
+2. There still might be issues with this feature when trying to use it (please inform us if so!).
```

Comment:
nit: we're expect an issue to be raised right (otherwise i imagine an email/tweet)